code/modules/mob/dead/observer/observer.dm

```diff
@@ -197,10 +197,9 @@ GLOBAL_VAR_INIT(observer_default_invisibility, INVISIBILITY_OBSERVER)
  * or you might end up with hair on a sprite that's not supposed to get it.
  * Hair will always update its dir, so if your sprite has no dirs the haircut will go all over the place.
  * |- Ricotez
+ * Note: This doesn't call parent because the base update_icon utilizes bitflags, while we use string icon states here.
  */
 /mob/dead/observer/update_icon(new_form)
```

Comment:
Really this should be fixed by adding new_form as a second parameter and keeping the bitflag parameter but not using it